newrelic_beego.go

```diff
@@ -45,21 +55,44 @@ func init() {
 	beego.InsertFilter("*", beego.BeforeRouter, StartTransaction, false)
 	beego.InsertFilter("*", beego.AfterExec, NameTransaction, false)
 	beego.InsertFilter("*", beego.FinishRouter, EndTransaction, false)
-	beego.Info("NewRelic agent start")
+	beego.Info("NewRelic agent started")
+}
+
+// parseSkipPaths gets string of comma separated paths
+// It returns a set of normalized paths
+func parseSkipPaths(pathsConfig string) map[string]bool {
+	paths := map[string]bool{}
+	splitPaths := strings.Split(pathsConfig, ",")
+
+	for _, path := range splitPaths {
+		formatted := strings.TrimSpace(path)
+		formatted = strings.ToLower(formatted)
+		if formatted != "" {
+			paths[formatted] = true
+		}
+	}
+
+	return paths
 }
 
+const newRelicTransaction = "newrelic_transaction"
+
 func StartTransaction(ctx *context.Context) {
+	if skipPaths[ctx.Request.URL.Path] {
```

Comment:
Impossible to have the exact path (will contain ids, envs etc.)
Should be loop on the map and check contains on each value

Reply:
Added support for wildcard prefix as well